### ISSUE TITLE
Is there a way to have consistant sentence case for context responses?

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#200]
Based on https://github.com/phetsims/gravity-force-lab/issues/155#issuecomment-552709648

@terracoda would like to discuss further the possibilities for a general solution that would help with easily producing sentence case for context responses.

Early investigations have happened in https://github.com/phetsims/molarity/issues/107

**Comment by zepumph** [comment]
I'm going to ask the developers if there are any ideas this Thursday. I'm not hopeful though.

**Comment by terracoda** [comment]
I think the only thing that we would have to check is the case where a general description parameter (i.e., one that is used in object response or in dynamic descriptions, and also in context responses).  When a general parameter starts a context response, we may need to double check it and potentially switch it to upper case.

**Comment by terracoda** [comment]
Hmm, I guess there could be similar situtations in the PDOM? Maybe the issue is more general to both the context responses and the PDOM content.

**Comment by zepumph** [comment]
After talking with developers, the current method of doing this is the recommended approach. If a capitalized and a lowercased string both needed, then two translatable strings must be provided. 
Here is a copy of the decision making:
![image](https://user-images.githubusercontent.com/6856943/69103797-db1cac80-0a12-11ea-8588-1e3120f0090e.png)

@terracoda I think this should be closed, please do so or let me know what else you would like to discuss.

**Comment by terracoda** [comment]
Thanks @zepumph and other devs for the discussion. I am good with this decision.
